binder/build.gradle

```diff
@@ -1,7 +1,6 @@
 plugins {
     id "maven-publish"
     id "com.android.library"
-    id "ru.vyarus.animalsniffer"
```

Comment:
Is this deleted just because there is no `signatures` configuration in the module, unrelated to gradle plugin update?

Reply:
See the PR description. I have a full paragraph about what is going on here.

Reply:
The short answer is that yes there are no signatures, but `animalsniffer` also requires the `java` plugin. So it is doubly misleading and did cause a failure at one point.